runner.electron/main.js

```diff
@@ -1,9 +1,16 @@
 var app = require('electron').app;
 var BrowserWindow = require('electron').BrowserWindow;
+var pkg = require('./package.json');
 
 var mainWindow = null;
 var opts = %OPTS%;
 
+if (pkg && Array.isArray(pkg.commandLineSwitches)) {
+	pkg.commandLineSwitches.forEach(function(cliSwitch) {
```

Comment:
i do not fully understand why you take `commandLineSwitches` from `package.json` instead of `opts` variable.

Reply:
Based on my reading of [index.js](index.js) the `opts` variable is populated by the `electronOpts`: https://github.com/lele85/karma-electron-launcher/blob/ef2dd61f0dd9a9974fb6ddf1667b40a519025a79/index.js#L40

Whereas `package.json` comes from the `options` key: https://github.com/lele85/karma-electron-launcher/blob/ef2dd61f0dd9a9974fb6ddf1667b40a519025a79/index.js#L16

The difference is the same `electronOpts` is applied to all electron launchers but you can have a different `options` object per custom electron launcher.

See the following example:

```
module.exports = function(config) {
  config.set({
    browsers: ['ElectronRealDevices', 'ElectronFakeDevices'],
    customLaunchers: {
      ElectronRealDevices: {
        base: 'Electron',
      },
      ElectronFakeDevices: {
        base: 'Electron',
        options: {
          commandLineSwitches: [
            '--use-fake-ui-for-media-stream',
            '--use-fake-device-for-media-stream',
          ],
        },
      },
    },
    electronOpts: {
      title: 'my window title'
    }
  });
};
```

The custom window title will be applied to both `ElectronRealDevices` and `ElectronFakeDevices` but the custom CLI switches will only be applied to `ElectronFakeDevices`. If we define the CLI switches in `electronOpts` then they will also be used in `ElectronRealDevices` which is undesirable.

The proper fix is probably to do to something like this in [runner.electron/main.js](runner.electron/main.js):

```
var opts = merge({}, %OPTS%, require('./package.json').electronOpts);
```

This will give more granular control of any electron option, not only `commandLineSwitches`. What do you think?

Reply:
Do you think we can avoid using `package.json` at all and use `electronOpts` from every launcher?
E.g. `var browserOptions = merge(true, defaultElectron, electronOpts || {}, args.electronOpts || {});`

Reply:
Sounds good to me. I'll implement this.